cloudferrylib/utils/__init__.py

```diff
@@ -11,7 +11,3 @@
 # implied.
 # See the License for the specific language governing permissions and#
 # limitations under the License.
```

Comment:
Well this seems to be unrelated to the purpose of the patch, plus it's not 100% clear whether this particular file needs to be copyrighted or not, according to [Apache FAQ](http://www.apache.org/legal/src-headers.html#faq-exceptions):

> Q: What files in an Apache release do not require a license header?
> A: A file without any degree of creativity in either its literal elements or its structure is not protected by copyright law; therefore, such a file does not require a license header. **If in doubt about the extent of the file's creativity, add the license header to the file**.

Since we're in doubt, let's keep the header there :)